ovip-0014.md

```diff
@@ -38,18 +38,25 @@ Initiator (Originator VASP)
 | Version     |                                      | `version`     | String       | 1..1  | fixed value `1.0`   |
 | Originator  | see [ovip-0013](https://github.com/OpenVASP/ovips/blob/master/ovip-0013.md#12-originator) (1.2) for elements | `originator`  |              | 1..1  |                     |
 | Beneficiary | see [ovip-0013](https://github.com/OpenVASP/ovips/blob/master/ovip-0013.md#13-beneficiary) (1.3) for elements | `beneficiary` |              | 1..1  |                     |
-| Transfer    |                                      | `transfer`    |              | 1..1  |                     |
-|             | Virtual asset type                   | `va`          | String       | 1..1  |                     |
-|             | Transfer type                        | `ttype`       | dt.TFR.ttype | 1..1  |                     |
-|             | Amount                               | `amount`      | Decimal      | 1..1  | 18 digits precision |
-| Comment     |                                      | `comment`     | String       | 0..1  |                     |
+| Transfer    |                                      | `transfer`    |                    | 1..1  |                     |
+|             | Virtual asset identifier             | `vaid`        | String             | 1..1  |                     |
```

Comment:
Not sure how much we care about backwards compat but this field could stay the same as it was before. In case you need to change it, I'd prefer `vaId` as name or even `virtualAssetId`.

Reply:
If backwards compatibility is a major concern at this point then you should go with va. But it really is an identifier (also was initially) so vaId (or virtualAssetIdentifier) would be appropriate. Its your call.

Reply:
Agree on CamelCase?

Reply:
As I said, I'm not sure if backwards compatibility is a major concern. Maybe @Felix0301 can share his thoughts?

Reply:
I don't think backwards compatibility should be the primary issue. But since we already had a field here with the same purpose, I don't see a good reason to rename - so keeping 'va' should be fine

Reply:
I don't think backwards compatibility should be the main concern as we won't have that anyway

Reply:
virtualAssetId sounds good to me